server/routes.go

```diff
@@ -889,7 +896,12 @@ func getKVData(digest string, verbose bool) (ggml.KV, error) {
 		}
 	}
 
-	return kv, nil
+	return kv, data.Tensors(), nil
+}
+
+func getKVData(digest string, verbose bool) (ggml.KV, error) {
```

Comment:
This is used in two places, which just us the kv value and err directly. Let's remove a layer of complexity and remove this function imo.